0.1.0/README.md

```diff
@@ -0,0 +1,25 @@
+# UGent Master Thesis Template
+
+This Typst template is made for a master thesis at UGent.
+The authors of this template are not affiliated with UGent.
+
+## Usage
+
+Run the following command in your terminal. The font path needs to be supplied as an argument if 
+the UGent fonts are not installed system-wide. This command will create the output at `main.pdf`.
+
+```shell
+typst watch main.typ --font-path fonts
+```
+
+## Features
+- styling of headings and page numbering, etc.
+- `#todo[your todo note]`: a to-do note that is displayed as a colored block in the output.
```

Comment:
Love the todo feature, but maybe put it in a different PR and diff against this one.

Reply:
✅